docs/usage_and_configuration/the_basics.md

```diff
@@ -0,0 +1,226 @@
+
+    --fast / --safe                 If --fast given, skip temporary sanity
+                                    checks. [default: --safe]
+
+    --include TEXT                  A regular expression that matches files and
+                                    directories that should be included on
+                                    recursive searches. An empty value means
+                                    all files are included regardless of the
+                                    name. Use forward slashes for directories
+                                    on all platforms (Windows, too). Exclusions
+                                    are calculated first, inclusions later.
+                                    [default: \.pyi?$]
+
+    --exclude TEXT                  A regular expression that matches files and
+                                    directories that should be excluded on
+                                    recursive searches. An empty value means no
+                                    paths are excluded. Use forward slashes for
+                                    directories on all platforms (Windows, too).
+                                    Exclusions are calculated first, inclusions
+                                    later.  [default: /(\.direnv|\.eggs|\.git|\.
+                                    hg|\.mypy_cache|\.nox|\.tox|\.venv|venv|\.svn|_bu
+                                    ild|buck-out|build|dist)/]
+
+    --extend-exclude TEXT           Like --exclude, but adds additional files
+                                    and directories on top of the excluded
+                                    ones (useful if you simply want to add to
+                                    the default).
+
+    --force-exclude TEXT            Like --exclude, but files and directories
+                                    matching this regex will be excluded even
+                                    when they are passed explicitly as
+                                    arguments.
+
+
+    --stdin-filename TEXT           The name of the file when passing it through
+                                    stdin. Useful to make sure Black will
+                                    respect --force-exclude option on some
+                                    editors that rely on using stdin.
+
+    -q, --quiet                     Don't emit non-error messages to stderr.
+                                    Errors are still emitted; silence those with
+                                    2>/dev/null.
+
+    -v, --verbose                   Also emit messages to stderr about files
+                                    that were not changed or were ignored due to
+                                    exclusion patterns.
+
+    --version                       Show the version and exit.
+    --config FILE                   Read configuration from FILE path.
+    -h, --help                      Show this message and exit.
+```
```

Comment:
Side note: I know it's nice, but I hate how these go out of date (And do for other projects). I've wondered if we can in a follow up PR create a CI action to dump `--help` to a file and include it somehow?

Reply:
Filed https://github.com/pypa/bandersnatch/issues/907 for bandersnatch to see if we can POC something there ...